app/controllers/api/v1/packages_controller.rb

```diff
@@ -225,6 +228,39 @@ def send_stock_item_response
         end
       end
 
+      def add_remove_item
+        render nothing: true, status: 204 and return if params[:quantity].to_i.zero?
+        response = Package::Operations.pack_or_unpack(
+                    container: Package.find(params[:id]),
+                    package: Package.find(params[:item_id]),
+                    quantity: params[:quantity].to_i, # quantity to pack or unpack
+                    location_id: params[:location_id],
+                    user_id: User.current_user.id,
+                    task: params[:task]
+                  )
+        if response[:success]
+          render json: { packages_inventories: response[:packages_inventory] }, status: 201
+        else
+          render json: { errors: response[:errors] }, status: 422
+        end
+      end
+
+      def contained_packages
+        entity = Package.find(params[:id])
```

Comment:
Should it be `find_by(id: params[:id])`, as on next we are verifying if `entity` is present or not.